service-providers/network-requester/src/main.rs

```diff
@@ -26,7 +26,7 @@ struct Run {
     #[clap(long)]
     open_proxy: bool,
 
-    /// Websocket port to bind to
+    /// Websocket port to bind to. Defaults to `network_defaults::DEFAULT_WEBSOCKET_LISTENING_PORT` (currently 1977)
```

Comment:
This will print that constant name verbatim in `--help`, not the value, just checking that it's intentional?